platform/core/src/DataSources/IWebApiDataSource.js

```diff
@@ -44,13 +45,20 @@ function create({
     series: {},
   };
 
+  const defaultStore = {
+    dicom: async naturalizedDataset => {
+      throw new Error(
+        'store.dicom(naturalizedDicom, StudyInstanceUID) not implemented for dataSource.'
+      );
+    },
+  };
+
   return {
     query: query || defaultQuery,
     retrieve: retrieve || defaultRetrieve,
+    store: store || defaultStore,
```

Comment:
We really need to discuss the shape of this API pre big release.
This follows the current pattern. Data sources are weird.

Reply:
Agreed. I at least chose not to call it stow as that is DICOMWeb specific.